### PR TITLE
Multiple fixes

### PR DESCRIPTION
**Pull request by gennadylaptev**

### CODE REVIEW

File: neurograph/train/train.py
Comment (maryjis) [inline neurograph/train/train.py:1]: Так не будет работать с  BolTLoss, он принимает на вход tuple ( logits,cls) и делает регуляризацию с cls

File: neurograph/unidata/base_dataset.py
Comment (maryjis) [inline neurograph/unidata/base_dataset.py:1]: Надо убрать  self.train  из get_cv_loaders,  get_test_loader  и заменить Subset на self.get_subset()

File: neurograph/unidata/dense.py
Comment (maryjis) [inline neurograph/unidata/dense.py:1]: Надо убрать self.train из get_cv_loaders, get_test_loader и заменить Subset на self.get_subset()

Comment (gennadylaptev) [reply neurograph/unidata/dense.py:1]: Лучше использовать `self.to_train()` и `self.to_test()`, чтобы если мы добавим новые аугментации выключать их все вместе
